openapiart/generator.py

```diff
@@ -114,6 +115,12 @@ def generate(self):
             common_content = fp.read()
             if re.search(r'def[\s+]api\(', common_content) is not None:
                 self._generated_top_level_factories.append('api')
+            if self._extension_prefix is None:
+                common_content = common_content.replace("\"__REPLACE_EXTENSION_PREFIX\"",
+                                                        "__name__")
+            else:
```

Comment:
if `self._extension_prefix` is not None, replace "__name__, ext" with "\"{}\", ext".format(self._extension_prefix)